confluent_osdeploy/ubuntu20.04/initramfs/scripts/init-premount/confluent

```diff
@@ -31,11 +31,33 @@ if [ -e /dev/disk/by-label/CNFLNT_IDNT ]; then
             MYGW=""
         fi
         MYNM=$(grep ^ipv4_netmask: $tcfg | awk '{print $2}')
-        for NICGUESS in $(ip link|grep LOWER_UP|grep -v LOOPBACK|cut -d ' ' -f 2 | sed -e 's/:$//'); do
+        for NICGUESS in `ls /sys/class/net/`; do
+            ip link set $NICGUESS up
```

Comment:
I'd be tempted to do this in one loop, then another follow up loop looking for link on any non-LOOPBACK, rather than trying each nic (including lo) for 20 seconds before moving on.  So I'd think:
for NICGUESS in `ls /sys/class/net/`; do
            ip link set $NICGUESS up
  done       
  retries=30
while retries; ip link |grep LOWER_UP|grep -v LOOPBACK', check result; retries--; done
Then if at least one LOWER_UP, non LOOPBACK, move on to the traditional loop for checking layer 3